File: app/src/main/java/com/example/megaport/mynews/Controllers/Activities/AboutActivity.java
package com.example.megaport.mynews.Controllers.Activities;

import android.content.Intent;
import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.ImageButton;
import com.example.megaport.mynews.R;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate( savedInstanceState );
        setContentView( R.layout.about_activity );

        ImageButton imageButton = findViewById( R.id.image_button_about );

        imageButton.setOnClickListener( view -> startActivity() );

    }

    private void startActivity(){
        Intent intent = new Intent( this, MainActivity.class );
        startActivity(intent);
    }
}

File: app/src/main/java/com/example/megaport/mynews/controllers/activities/AboutActivity.kt
package com.example.megaport.mynews.controllers.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.megaport.mynews.R
import kotlinx.android.synthetic.main.about_activity.*

class AboutActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.about_activity)

        image_button_about.setOnClickListener{ finish() }
    }
}
